src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall(Lottery ball) {
+        int number = new Random().nextInt(100);
```

Comment:
Creating a new instance of `Random` every time the method is called is inefficient. Consider creating a single `Random` instance as a private class field and reusing it.